### PR TITLE
add formula for Domoticz

### PR DESCRIPTION
**Pull request by bjpbakker**
As mentioned in PR #36789 on Homebrew/homebrew, by lack of release
tarballs this formula belongs in Homebrew/homebrew-head-only.

Feedback on the original PR has been fixed.

### CODE REVIEW

**Comment by L2G** [comment]
Looks like @BrewTestBot got stuck on this one. (But I'm not versed in the ways of the Brew test bot.)

File: domoticz.rb [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      <false/>
+      <key>Label</key>
+      <string>#{plist_name}</string>
+      <key>ProgramArguments</key>
+      <array>
+        <string>#{prefix}/opt/domoticz/domoticz</string>
+        <string>-www</string>
+        <string>8083</string>
+        <string>-log</string>
+        <string>/var/log/domoticz.log</string>
```
Comment (DomT4) [inline domoticz.rb]: This needs to be `#{var}` instead of just `var`.

File: domoticz.rb [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        <string>0</string>
+      </array>
+      <key>OnDemand</key>
+      <false/>
+      <key>KeepAlive</key>
+      <true/>
+      <key>RunAtLoad</key>
+      <true/>
+      <key>WorkingDirectory</key>
+      <string>#{prefix}/opt/domoticz</string>
```
Comment (DomT4) [inline domoticz.rb]: There's an `opt` directory inside of the formula's `prefix`?

Comment (DomT4) [inline domoticz.rb]: I think you may be confusing `HOMEBREW_PREFIX` with `prefix` here?

**Comment by DomT4** [comment]
@BrewTestBot test this please

**Comment by DomT4** [comment]
Some other things the bot highlighted need fixing as well:

```
==> brew style domoticz

1 file inspected, no offenses detected
Error: 3 problems in 1 formula

==> audit problems
homebrew/head-only/domoticz:
 * Formula should have a desc (Description).
 * 17: Trailing whitespace was found
 * 18: Trailing whitespace was found

==> brew test --verbose Homebrew/head-only/domoticz			 FAILED
Testing homebrew/head-only/domoticz
==> /usr/local/Cellar/domoticz/HEAD/opt/domoticz/domoticz -h
libc++abi.dylib: terminating with uncaught exception of type boost::exception_detail::clone_impl<boost::exception_detail::error_info_injector<boost::lock_error> >: boost: mutex lock failed in pthread_mutex_lock: Invalid argument
Error: homebrew/head-only/domoticz: failed
<0> expected but was
<nil>.
```

**Comment by DomT4** [comment]
Closing as dead. Please open a new updated PR if you'd like to.

**Comment by saerdnaer** [comment]
Just if someone is trying to build domoticz manually or with the formula which was started here. Have a look at http://domoticz.com/wiki/Mac_OS_X#Homebrew , there is more up to date information

**Pull request closed without merging**